cogs/search.py

```diff
@@ -0,0 +1,297 @@
+                await ctx.send("**Sorry!** That query included language we cannot accept in a non-NSFW channel. Please try again in an NSFW channel.")
+
+    @command()
+    async def anime(self, ctx: Context, *, query: str) -> None:
+        """Look up anime information."""
+        base = "https://kitsu.io/api/edge/"
+
+        # Handling
+        async with ctx.typing():
+            async with aiohttp.ClientSession() as session:
+                async with session.get(base + "anime", params={"filter[text]": query}) as resp:
+
+                    resp = await resp.json()
+                    resp = resp["data"]
+
+                    query = utils.escape_mentions(query)
+                    query = utils.escape_markdown(query)
+
+                    if not resp:
+                        return await ctx.send(f"No results for `{query}`.")
+
+                    anime = resp[0]
+                    title = f'{anime["attributes"]["canonicalTitle"]}'
+                    anime_id = anime["id"]
+                    url = f"https://kitsu.io/anime/{anime_id}"
+                    thing = "" if not anime["attributes"]["endDate"] else f' to {anime["attributes"]["endDate"]}'
+
+                    embed = Embed(title=f"{title}", color=ctx.author.color, rl=url)
+                    embed.description = anime["attributes"]["synopsis"][0:425] + "..."
+                    embed.add_field(name="Average Rating", value=anime["attributes"]["averageRating"])
+                    embed.add_field(name="Popularity Rank", value=anime["attributes"]["popularityRank"])
+                    embed.add_field(name="Age Rating", value=anime["attributes"]["ageRating"])
+                    embed.add_field(name="Status", value=anime["attributes"]["status"])
+
+                    embed.add_field(name="Aired", value=f"{anime['attributes']['startDate']}{thing}")
+                    embed.add_field(name="Episodes", value=anime["attributes"]["episodeCount"])
+                    embed.add_field(name="Type", value=anime["attributes"]["showType"])
+                    embed.set_thumbnail(url=anime["attributes"]["posterImage"]["original"])
+                    embed.set_footer(text=f"Requested by {ctx.author.name} | Powered by kitsu.io", icon_url=ctx.author.avatar_url_as(format="png"))
+                    try:
+
+                        await ctx.send(f"**{title}** - <{url}>", embed=embed)
+
+                    except Exception:
+                        aired = f"{anime['attributes']['startDate']}{thing}"
+                        template = f"""```
+        url: {url}
+        Title: {title}
+        Average Rating: {anime["attributes"]["averageRating"]}
+        Popularity Rank: {anime["attributes"]["popularityRank"]}
+        Age Rating: {anime["attributes"]["ageRating"]}
+        Status: {anime["attributes"]["status"]}
+        Aired: {aired}
+        Type: {anime['attributes']["showType"]}
+        Powered by kitsu.io```"""
```

Comment:
Fix this string indentation, note that you'll need to import `textwrap.dedent`.
```suggestion
                        template = textwrap.dedent(
                            f"""
                            ```
                            url: {url}
                            Title: {title}
                            Average Rating: {anime['attributes']['averageRating']}
                            Popularity Rank: {anime['attributes']['popularityRank']}
                            Age Rating: {anime['attributes']['ageRating']}
                            Status: {anime['attributes']['status']}
                            Aired: {aired}
                            Type: {anime['attributes']['showType']}
                            Powered by kitsu.io
                            ```
                            """
                        )
```